cpp/ql/src/semmle/code/cpp/ir/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -251,27 +251,28 @@ private predicate getWrittenField(Instruction instr, Field f, Class c) {
 }
 
 private predicate fieldStoreStepChi(Node node1, FieldContent f, PostUpdateNode node2) {
-  exists(StoreInstruction store, ChiInstruction chi |
-    node1.asInstruction() = store and
+  exists(StoreValueOperand operand, ChiInstruction chi |
+    node1.asOperand() = operand and
```

Comment:
Same questions about partial defs.

Reply:
You're right. It shouldn't actually be a `StoreValueOperand`, but rather a `ChiPartialOperand`. From `simpleOperandLocalFlowStep` we get flow from the result of the store instruction to any total exact overlapping use. So if we add a `storeStep` from the partial chi operand to its chi instruction we get `store -> chi partial operand -> chi instruction` flow.

Reply:
Fixed in 3c778868f8a5142a6f1862167226b0609f7c1658. Surprisingly, it had zero effect on the path explanations in the library-test files (but a few in the query-test files).